src/java/src/main/java/com/dnanexus/DXWorkflow.java

```diff
@@ -239,4 +243,101 @@ public Describe getCachedDescribe() {
         return ExecutableRunner.getWorkflowRunnerWithEnvironment(this.getId(), this.env);
     }
 
+    /**
+     * A workflow stage
+     *
+     */
+    public static class DXStage {
+        private String ID;
+
+        DXStage(String ID) {
+            this.ID = ID;
+        }
+
+        public String getId() {
+            return ID;
+        }
+
+        /**
+         * Create a link to an output field. This is used in workflows, to
+         * link results between stages.
+         *
+         * @param fieldName  name of an output field
+         *
+         * @return JSON representation of a link. Can be used as an input to a workflow stage.
+         */
+        public ObjectNode getOutputReference(String  fieldName) {
```

Comment:
Maybe make this declare return type `JsonNode` to be more similar to `getDXLink`...

Also, `fieldName` => `outputName`, and similarly `inputName` below